doc/design/s3-backup.md

```diff
@@ -0,0 +1,92 @@
+- **Glacier Deep Archive requires 12-48 hours to retrieve data:** the exact amount of time depends on the retrieval mechanism (see [AWS docs](https://aws.amazon.com/s3/storage-classes/glacier/)), but either way, instantaneous recovery is not possible. However, this latency is acceptable for purposes of catastrophic data recovery; it also indicates why the trailing delete mechanism continues to be valuable (it offers a much faster "data recovery" in the cases where it can be used).
+
+## Cost
+
+**Below are the additional costs introduced by this backup feature** for a 1 PB primary bucket (assuming both the primary bucket and backup bucket are in us-east-2). All of this information was gathered from the "Storage & requests", "Data transfer", and "Replication" tabs on [https://aws.amazon.com/s3/pricing/](https://aws.amazon.com/s3/pricing/).
+
+**Storage Costs** (backup bucket in us-east-2):
+
+- Glacier Deep Archive storage: ~$0.00099/GB/month
+    - 1 PB = 1,000 TB × $0.99/TB = **$990/month**
+
+**Data Transfer Costs**:
+
+- Same-region data transfer between S3 buckets is free
+
+**Retrieval Costs** (only incurred when disaster recovery is needed):
+
+- Glacier Deep Archive retrieval:
+    - $0.02/GB (standard, 12-hour retrieval)
+    - $0.0025/GB (bulk retrieval, can take up to 48 hours)
+
+Imagining that the entire primary bucket was destroyed (which is not the
+expected scale of data loss, but useful as a worst-case analysis), then the cost
+to restore from backup would be
+
+$`1\ \rm{PB} \times \frac{1000 \ \rm{TB}}{\rm{PB}} \times \frac{1000 \ \rm{GB}}{\rm{TB}} \times \frac{\$0.0025/mo}{\rm{GB}} = \$2500`$.
+
+### Future Costs
+
+The DANDI Archive is expecting a ramp-up in data volume of 1 PB of new data over each of the next five years, culminating in a total of 6PB.
+
+Scaling up the previous analysis means that the monthly costs will be projected to rise to a total of **~$5,900/month** once all of that data is seated.
+The worst-case disaster recovery cost would similarly scale up to a total of **~$16,000**.
+
+An open question is whether the AWS Open Data Sponsorship program would cover the marginal costs of backup. A quick estimate shows that once all 5 PB has been uploaded, the expected bucket cost for the primary bucket (i.e., what the AWS Open Data Sponsorship program covers already, excluding backup) will be:
+
+$$
+6\ \rm{PB} \times \frac{1000\ \rm{TB}}{\rm{PB}} \times \frac{1000\ \rm{GB}}{\rm{TB}} \times \frac{\$0.021/mo}{\rm{GB}} \approxeq \$126000/mo
+$$
+
+while the associated backup costs would represent only an additional $`\$5900 / \$126000 \approxeq 4.6\%`$ of the cost of the storage itself.
+To help provide a significant level of safety to an important dataset, AWS may be willing to cover such a low marginal cost.
```

Comment:
@satra The design here looks good, but the main open question and blocker is cost.

Would the AWS Open Data program be open to covering the back up storage costs as well (~$12,000/year for a 1 PB backup, and ~$72,000/year for the projected 6 PB backup)? See the doc for more details.